### DISCUSSION TITLE
Github query in Shortcuts

### DISCUSSION THREAD

**Discussion by danirzv** [community/community#86784] [Mobile]
### Select Topic Area

Bug

## Action Performed:

- Open **Github** app on **Android**
- Go to **Home** tab
- Scroll to **Shortcuts**
- Click on triple-dots
- Click on **Create Shortcut**

## Expected Result:
Being able to use Github queries for example:
`is:open is:issue label:"help candidate","help wanted" -label:area-blazor `
so I can find issues with lablel "help candidate" OR "help wanted" AND NOT "area-blazor"

## Actual Result:
Spent some time to find out I only can filter labels as AND between them and no option for NOT or OR

## User impact:
Shortcut part is not as efficient as it easily could be, so it's not usable for me at the moment

## Suggested resolution:
let user to input query as web interface on shortcuts

## Platforms:
Github on Android

**Comment by mattruggio** [comment]
👋 @danirzv! Thanks for identifying this parity issue between web and mobile! I went through and tried a few comparable queries/shortcuts out and definitely was able to reproduce this. Interesting enough, the query _does_ appear to function correctly on mobile when copying/pasting the raw query into the search field. Problems do seem to arise once the query is transformed into a shortcut. I brought this to the attention of our client engineering team and will definitely create an internal issue to track this but cannot speculate on prioritization of it.

**Reply by danirzv** [reply]
Any update about this issue?

**Comment by kareimk333** [comment]
![Uploading Screenshot_2024-02-28-23-50-29-76_a23b203fd3aafc6dcb84e438dda678b6.jpg…]()

**Comment by kareimk333** [comment]
حنين صبري

**Reply by kareimk333** [reply]
https://www.facebook.com/profile.php?id=61550797642992&mibextid=ZbWKwL

**Comment by kareimk333** [comment]
![Uploading Screenshot_2024-02-28-23-50-29-76_a23b203fd3aafc6dcb84e438dda678b6.jpg…]()
